backend/src/lib/controllers/user-ctrl.ts

```diff
@@ -19,6 +20,57 @@ async function getUsers(req: Request, res: Response) {
   }
 }
 
+async function addUsers(req: Request, res: Response) {
+  const user = AuthService.getCurrentUser(req);
+
+  if (!user) {
+    res.status(401);
+    return res.send("Unauthorized");
+  }
+
+  const { role } = req.body;
+  const { emails } = req.query;
```

Comment:
Why are the email addresses in the query params and not in the body? This might be a security concern because query parameters get saved in the browsers history. 

Ref: https://owasp.org/www-community/vulnerabilities/Information_exposure_through_query_strings_in_url